tests/test_rapid_tests.py

```diff
@@ -155,6 +156,43 @@ def test_sample_test_outcome_with_sensitivity(params):
     assert np.isclose(is_tested_positive.mean(), sensitivity, atol=1e-3)
 
 
+@pytest.mark.unit
+def test_sample_test_outcome_with_sensitivity_time_dependent(params):
+    n_individuals = 10_000_000
+
+    receives_rapid_test = np.ones(n_individuals).astype(bool)
+    cd_infectious_true = np.random.choice([0, -1, -2], n_individuals, True)
```

Comment:
We should have one strictly positive entry here. And have asserts for that case as well as the -2 case.